### DISCUSSION TITLE
为专业版提供自定义登录页面的选项

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [halo-dev/halo#7018] [Ideas]
### 前置条件

- [x] 已经在 [Issues](https://github.com/halo-dev/halo/issues) 列表中搜索了相关问题。
- [x] 这是和 Halo 相关的特性，如果是非项目本身的问题，建议在 [Discussions](https://github.com/halo-dev/halo/discussions) 提交。
- [x] 如果是插件和主题特性建议，请在对应的插件和主题仓库提交。

### 你当前使用的版本

2.20.8

### 描述一下此特性

在Halo 2.20LTS中支持了自定义登录页面,但需要手动修改源码,这对于生产环境来说要是改错了什么东西可就坏事了。
故希望可以给专业版用户提供一个自定义登录页面的设置(在`品牌信息`选项里加),可以自定义登录背景图片、按钮颜色什么的。

### 附加信息

这个issue的想法只是想方便那些不愿意整那么麻烦的人和一些一点代码都看不懂的小白。
——只是加了一种自定义方式,并不是让把之前的路堵死~~来让用户喷凌霞想钱想疯了~~。
至于给只给专业版开放是因为品牌信息这一栏所有内容都要激活许可证才能用,所以才这么想。

**Comment by w-flac** [comment]
这种自定义设置…不是由主题实现么？这个跟halo有啥关系…
